### ISSUE TITLE
Generators create MiniTest tests after RSpec install

### ISSUE THREAD

**Issue by johnpitchko** [rspec/rspec-rails#2615]
## What Ruby, Rails and RSpec versions are you using?

Ruby version: 3.0.4
Rails version: 7.0.3.1
RSpec version: 3.11
RSpec Rails version: 5.1.2

## Observed behaviour

1. Added `rspec-rails` gem to Gemfile and installed gem with `bundle`.
2. Ran `rails generate rspec:install` to install RSpec.
3. Ran 'rails g migration <model name> <field name>:<type>...`

Generator created unit tests with MiniTest (see screenshot).

<img width="570" alt="image" src="https://user-images.githubusercontent.com/7190681/184576277-20e8a1e7-a181-4af0-8bd9-3191fdab6d93.png">

## Expected behaviour

Generator creates (RSpec) specs instead of (MiniTest) tests.

*Note:* once I added `config.generators.test_framework = :rspec` to `config/application.rb`, the generator created specs when I generated a model.

## Can you provide an example app?

Happy to do so if requested.

**Comment by JonRowe** [comment]
What does your `Gemfile` line look like? The default should be set by the railtie but you'd need to have it loaded it both dev and test, and not surpressed the require... Additionally to turn off mini test you may have to have generated the project with `--no-test` before running `rspec:install`

**Comment by johnpitchko** [comment]
To verify, I created a brand new Rails app and followed the install steps from the README. Worked fine there; generating a model did create the spec files as expected.

I looked at the Gemfile for my project and I noticed that I included the `rspec-rails` gem in `group :test` instead of in `group :development, :test`.

I then created _another_ Rails project and added `rspec-rails` to the `group :test`. When I generated a model, MiniTests were created. So, yes, where you group the gem in `Gemfile` _absolutely_ affects what tests are generated! Was not expecting that 😆 

So this was my bad - I didn't follow the instructions correctly. As mentioned above, had no idea that gem grouping would affect the generators. And now I just read this line:

> (Adding it to the :development group is not strictly necessary, but without it, generators and rake tasks must be preceded by RAILS_ENV=test.)

Thanks for the quick response; I'm closing this issue and feeling rather ashamed 😄
